lib/LLVMIRCodeGen/BundleSaver.cpp

```diff
@@ -450,6 +457,66 @@ void BundleSaver::emitSymbolTable() {
                            irgen_->getBundleName() + "SymbolTable");
 }
 
+void BundleSaver::addBundleObjects(
```

Comment:
So, you try to produce a static library archive from the original object file generated by Glow and some additional object files, right?

Reply:
That is correct.